lib/src/app/app.dart

```diff
@@ -86,4 +87,25 @@ class AppClient {
     final stream = response.map((event) => event.logs);
     return stream.asBroadcastStream(onCancel: (_) => response.cancel());
   }
+
+  /// List the [Authorization]s available for the currently authenticated user
+  Future<List<Authorization>> listAuthorizations(String organizationId, {List<String> resourceIds = const []}) async {
+    final request = ListAuthorizationsRequest()
+      ..organizationId = organizationId
+      ..resourceIds.addAll(resourceIds);
```

Comment:
[`..`](https://dart.dev/language/operators#cascade-notation) is neat a language feature

Reply:
Ya SUPER useful in not having to create temporary objects for assignment